src/test/java/seedu/trackermon/model/UserPrefsTest.java

```diff
@@ -1,21 +1,21 @@
-//package seedu.trackermon.model;
-//
-//import static seedu.trackermon.testutil.Assert.assertThrows;
-//
-//import org.junit.jupiter.api.Test;
-//
-//public class UserPrefsTest {
-//
-//    @Test
-//    public void setGuiSettings_nullGuiSettings_throwsNullPointerException() {
-//        UserPrefs userPref = new UserPrefs();
-//        assertThrows(NullPointerException.class, () -> userPref.setGuiSettings(null));
-//    }
-//
-//    @Test
-//    public void setAddressBookFilePath_nullPath_throwsNullPointerException() {
-//        UserPrefs userPrefs = new UserPrefs();
-//        assertThrows(NullPointerException.class, () -> userPrefs.setAddressBookFilePath(null));
-//    }
-//
-//}
+package seedu.trackermon.model;
+
+import static seedu.trackermon.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class UserPrefsTest {
```

Comment:
Please add header comment?

Reply:
i opened a new issue for this, you can go there and discuss about who is going update what.